### PR TITLE
feat(insights): Add api to break down alert triggers by alert rule (WOR-1518)

### PR DESCRIPTION
**Pull request by wedamija**
This adds in an api to give us a breakdown of alert rules that fired the most in the last week, and
gives us average stats over the last 8 weeks as a comparison. Supports pagination.

### CODE REVIEW

**Review by scttcper — APPROVED** [review]
lgtm

File: src/sentry/api/endpoints/team_alerts_triggered.py [hunk 1]
```diff
@@ -61,3 +66,85 @@ def get(self, request: Request, team) -> Response:
             current_day += timedelta(days=1)
 
         return Response(counts)
+
+
+class TriggeredAlertRuleSerializer(AlertRuleSerializer):
+    WEEKS_TO_FETCH = 8
```
Comment (scttcper) [inline src/sentry/api/endpoints/team_alerts_triggered.py:72]: we might have to get this from the query parameter?

Comment (wedamija) [reply src/sentry/api/endpoints/team_alerts_triggered.py:72]: Right, I forgot that this is configurable

Comment (scttcper) [reply src/sentry/api/endpoints/team_alerts_triggered.py:72]: might also be okay to merge this and follow up with that

**Review by taylangocmen — APPROVED** [review]
Other than adding `statsPeriod` to the endpoint per @scttcper 's comment lgtm as well

**Pull request merged**